ui/app/routes/vault/cluster/secrets/backend/credentials.js

```diff
@@ -1,15 +1,28 @@
 import { resolve } from 'rsvp';
 import Route from '@ember/routing/route';
+import { getOwner } from '@ember/application';
+import { inject as service } from '@ember/service';
 
 const SUPPORTED_DYNAMIC_BACKENDS = ['ssh', 'aws', 'pki'];
 
 export default Route.extend({
   templateName: 'vault/cluster/secrets/backend/credentials',
+  pathHelp: service('path-help'),
 
   backendModel() {
     return this.modelFor('vault.cluster.secrets.backend');
   },
 
+  beforeModel() {
+    const { backend } = this.paramsFor('vault.cluster.secrets.backend');
+    if (backend != 'ssh') {
```

Comment:
maybe add a comment here that we're only doing openAPI for SSH credentials for now? PKI later?